packages/grafana-ui/src/components/Table/TableCell.tsx

```diff
@@ -48,6 +48,7 @@ export const TableCell = ({
       // justify-content flex-end is not compatible with cellLink overflow; use direction instead
```

Comment:
do we still need `!field.config.unit` check with this change?

Reply:
testing with this repro dashboard is seems that this check is no longer necessary
[debug-Panel Title-2024-07-22 15_02_28.json.txt](https://github.com/user-attachments/files/16340149/debug-Panel.Title-2024-07-22.15_02_28.json.txt)

Reply:
I concur!